_config.yml

```diff
@@ -33,14 +33,21 @@ include:
   - _pages
 
 contentful:
+  config:
+    sites: "www.crossroads.net"
```

Comment:
This is not an accurate use of this configuration value. Please review the docs and let me know if you have questions. https://github.com/crdschurch/jekyll-contentful#content-distribution